__tests__/contact-spec.js

```diff
@@ -100,28 +102,57 @@ describe('Contact', () => {
       testContext.connection.request = jest.fn(() => {
         const contactJSON = {
           id: '1257',
+          object: 'contact',
+          account_id: '1234',
           given_name: 'John',
           middle_name: 'Jacob',
           surname: 'Jingleheimer Schmidt',
           suffix: 'II',
+          nickname: 'John',
+          birthday: '2019-07-01',
+          company_name: 'Life',
           job_title: 'artist',
-          phone_numbers: [{'type': 'mobile', 'number': '555-444-3333'}],
+          manager_name: 'April',
+          office_location: 'SF, CA, USA',
+          notes: 'lalala',
+          picture_url: 'example.com',
+          emails: [{'type': 'work', 'email': 'john@test.com'}],
+          im_addresses: [{'type': 'yahoo', 'im_address': 'jjj'}],
           physical_addresses: [{'type': 'home', 'city': 'Boston'}],
+          phone_numbers: [{'type': 'mobile', 'number': '555-444-3333'}],
+          web_pages: [{'type': 'blog', 'url': 'johnblogs.com'}],
+          groups: [{id: '123', 'object': 'contact', 'account_id': '1234', 'name': 'Fam', 'path': 'Fam'}],
+          source: 'Contacts',
```

Comment:
source is either `address_book` or `inbox`

Reply:
oooh good catch. I updated to `inbox`